fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFab.kt

```diff
@@ -81,22 +74,14 @@ fun TabsTrayFab(
         }
     }
 
-    Box(
-        Modifier
-            .fillMaxSize()
-            .semantics { invisibleToUser() },
-    ) {
-        if (isInNormalMode) {
-            FloatingActionButton(
-                icon = icon,
-                modifier = Modifier
-                    .align(Alignment.BottomEnd)
-                    .padding(16.dp),
-                contentDescription = contentDescription,
-                label = label,
-                onClick = onClick,
-            )
-        }
+    if (isInNormalMode) {
+        FloatingActionButton(
+            icon = icon,
+            modifier = Modifier.padding(16.dp),
```

Comment:
This is a misstep from my PR, but could we also change this modifier to apply padding only for `bottom` and `end`? this way we lower the position of the `Undo` snackbar and achieve parity with the xml version.

Reply:
For sure! Thanks for catching this!